release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -144,6 +144,451 @@ io:println(nameText); // "DanBrown"
 
 #### Standard Library
 
+##### HTTP Package Updates
+
+Change the return type of client method to depend on `targetType` arg. The default targetType is `http:Response`
+```ballerina 
+http:Client myClient = check new("http://localhost:9090”);
+http:Response v = check myClient->post("/backend/getResponse", "want response");
+json p = check myClient->post("/backend/getJson", "want json", targetType = json);
+xml q = check myClient->post("/backend/getXml", "want xml", targetType = xml);
+```
+
+Introduce header map as an optional argument for Non-entity body client remote method(GET, HEAD, OPTIONS) 
+```ballerina
+http:Client myClient = check new("http://localhost:9090”);
+map<string|string[]> accHeaders = {};
+var response = myclient->get("/some/endpoint", accHeaders);
+```
+
+Introduce header map and media type as optional arguments for Entity body client remote method(POST, PUT, PATCH, DELETE, EXECUTE)
+```ballerina
+http:Client myClient = check new("http://localhost:9090”);
+json payload = {}; 
+map<string|string[]> accHeaders = {};
+var response = myclient->post("/some/endpoint", payload, headers = accHeaders);
+```
+
+Improve the data types of outbound request/response payloads which can be set directly.  
+```ballerina
+type RequestMessage Request|string|xml|json|byte[]|int|float|decimal|boolean|map<json>|table<map<json>>|
+                      (map<json>|table<map<json>>)[]|mime:Entity[]|stream<byte[], io:Error>|();
```

Comment:
Not related to the release note itself, but `json` represents pretty much everything else, the only additional type via `json` is `json[]`.

Can we not do
```suggestion
type RequestMessage Request|xml|json|table<map<json>>|table<map<json>>[]|mime:Entity[]|stream<byte[], io:Error>;
```

or if we want to have the different types explicitly
```suggestion
type RequestMessage Request|string|xml|json[]|byte[]|int|float|decimal|boolean|map<json>|table<map<json>>|
                      table<map<json>>[]|mime:Entity[]|stream<byte[], io:Error>|();
```